src/main.js

```diff
@@ -220,10 +220,6 @@ function codeReviewFeatures(config) {
         setLineLengthLimit(config.lineLengthLimit)
     }
 
-    if (config.ignoreWhitespace) {
-        ignoreWhitespaceInit()
-    }
-
```

Comment:
The goal behind putting this here was so that this feature also worked when viewing individual commits.

If this is causing issues for new/edit PRs, i think we should handle that case explicitly here instead so that we don't lose this functionality on those other pages.

What do you think?

Reply:
@jwallet